docker-app/qfieldcloud/core/serializers.py

```diff
@@ -523,3 +532,106 @@ class LatestPackageSerializer(serializers.Serializer):
     package_id = serializers.UUIDField()
     packaged_at = serializers.DateTimeField()
     data_last_updated_at = serializers.DateTimeField()
+
+
+class TeamSerializer(serializers.ModelSerializer):
+    organization = serializers.CharField(source='team_organization.username', read_only=True)
+    
+    class Meta:
+        model = Team
+        fields = ('username', 'organization')
+
+class TeamMemberSerializer(serializers.ModelSerializer):
+    member = serializers.StringRelatedField()  
+    
+    class Meta:
+        model = TeamMember
+        fields = ('team', 'member')
+
+class AddMemberSerializer(serializers.Serializer):
+    username = serializers.CharField(required=True)
+    email = serializers.EmailField(required=False, allow_blank=True)
+
+    def validate(self, data):
+        username = data.get('username')
+        user = self._validate_user_exists(username)
```

Comment:
This should also work with email too.


Basically, check if the user is a member of the organization already. That's all.

Reply:
Done